test/llvm_ir_correct/invalid_public_entity_error.f90

```diff
@@ -0,0 +1,33 @@
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+! RUN: %flang -S -emit-llvm %s
+
+MODULE dbcsr_operations
```

Comment:
I did a verification and it seems #966 does not fix this test. I got the following error:
```
F90-S-0084-Illegal use of symbol dbcsr_norm - not public entity of module
```

Reply:
Not sure what is the issue. For me both the testcases work with the patch. Since the patch is already merged, could you please test these testcases with tip of the trunk sources?

Reply:
My mistake, I was using a downstream version. Verified with tip of the trunk and things work fine.